mslib/index.py

```diff
@@ -104,6 +106,7 @@ def get_topmenu():
              ((url_for('about'), 'About'),
               (url_for('install'), 'Install'),
               (url_for('help'), 'Help'),
+              (url_for('plots'), 'Plots'),
```

Comment:
'Help' should be the last option in the list

Reply:
Done